02_Methodiek.Rmd

```diff
@@ -108,36 +126,48 @@ We beschouwen een meetpunt als geschikt voor het berekenen van een droogte-indic
 
 - De tijdreeks omspant binnen de periode _2001_ en _2016_ minstens _5 jaar_
 - Deze tijdreeks is _maximaal 2 opeenvolgende jaren_ onderbroken
-- Deze tijdreeks bevat _minstens 5 (hydrologische) jaren_ waarvoor een lg3[^1] kan berekend worden.
+- Deze tijdreeks bevat _minstens 5 (hydrologische) jaren_ waarvoor een lg3[^2] kan berekend worden.
 
 Het is belangrijk te vermelden dat deze basiskwaliteitscriteria vooral dienen voor een eerste screening van de meetpunten op basis van actueel beschikbare feitelijke grondwatermetingen. 
-De eigenlijke kwaliteit van de tijdreeks wordt bepaald door een tijdreeksanalyse (zie \@ref(tijdreeksanalyse). 
+
+De eigenlijke kwaliteit van de tijdreeks wordt bepaald door een tijdreeksanalyse (zie \@ref(tijdreeksanalyse)). 
 
 Een meetpunt kan op basis van de actueel beschikbare gegevens ongeschikt zijn, omdat de gegevens niet toelaten om deze indicator voldoende betrouwbaar te berekenen. 
 
 Nemen we ook het kwaliteitsaspect mee in de beoordeling, dan wordt de categorie-indeling van rastercellen complexer. 
-We kunnen niet alleen meerdere subcategorieën onderscheiden, maar een rastercel zal ook tot meerdere categorieën kunnen behoren.
+
+We kunnen niet alleen meerdere subcategorieën onderscheiden, de behandeling van de meetpunten in een rastercel wordt ook minder éénduidig.
+In het geval dat alle peilbuislocaties gelijkwaardig waren, bevatten alleen rastercellen van categorie 1 twee soorten meetpunten.
+Maken we een onderscheid in kwaliteit van de meetreeksen, dan zijn er meer toestanden mogelijk.
+We lichten dit toe aan de hand van een tweetal voorbeelden.
+
 Voorbeeld 1: in een rastercel zijn drie meetpunten gewenst. 
 Er zijn momenteel maar twee peilbuizen beschikbaar, één met een goede tijdreeks en één met een slechte. 
-De rastercel zal dan behoren tot categorie 1A: er dient nog een meetlocatie bij gezocht te worden, tot categorie 1B: de peilbuis met een slechte tijdreeks komt in de wachtrij en tot categorie 2: de peilbuis met een goede tijdreeks wordt sowieso geselecteerd voor onmiddellijke ingebruikname.
+De drie locaties vragen elk een andere benadering. 
+Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest presentatieve keuze. 
```

Comment:
```suggestion
Voor het meetpunt zonder peilbuis, moet op zoek gegaan worden naar de meest representatieve keuze. 
```

Reply:
hier en ook elders: 'meetpunt zonder peilbuis' lijkt te suggereren dat je de locatie ervan al kent ('meetpunt'). Maar is vermoedelijk niet zo. Misschien duidelijker om te spreken van een 'nog ontbrekend meetpunt', ...?